index.d.ts

```diff
@@ -181,6 +181,14 @@ export interface GetWindowBoundsCenteredOptions {
 	```
 	*/
 	readonly size?: Size;
+
+	/**
+	Use the full display size when calculating the position, rather than just the workable area.
```

Comment:
The description here and the readme should be the same.

I think it's useful to explain what the "workable" area is.